app/controllers/admin/employees_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Admin
+  class EmployeesController < AdminController
+    before_action :find_employee, only: %i[show edit update destroy]
+
+    # GET /admin/employees
+    def index
+      @employees = Employee.all
+    end
+
+    # GET /admin/employees/1
+    def show; end
+
+    # GET /admin/employees/1/edit
+    def edit; end
+
+    # PATCH/PUT /admin/employees/1
+    def update
+      params[:employee].delete(:password) if params[:employee][:password].blank?
```

Comment:
check out `params.compact`

Reply:
OK. I've read about it. Do you suggest using this method instead of `delete` or do you have something else in mind?

Reply:
exactly - it is designed to remove params, that are `nil`